pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/bestpractices/PositionLiteralsFirstInCaseInsensitiveComparisonsRule.java

```diff
@@ -4,6 +4,7 @@
 
 package net.sourceforge.pmd.lang.java.rule.bestpractices;
 
+@Deprecated
```

Comment:
We should add a javadoc note to mention, that the class has been replaced by LiteralsFirstInComparisons. I'll do that, when merging.